common-content/en/module/tools/shell-pipelines/index.md

```diff
@@ -5,11 +5,38 @@ time = 30
 facilitation = false
 emoji= "💻"
 [objectives]
-    1="Count the occurences of different lines within a file using sort and uniq"
-    2="List the files in a directory which contain an upper-case letter in their name with ls and grep"
-    3="Count the number of files in a directory which contain an upper-case letter in their name with ls, grep, and wc"
-    4="Replace all occurrences of one character with another using tr"
-    5="TODO: Pipe the output of jq to something interesting"
+    1="Describe what a shell pipeline is."
+    2="Explain why we use shell pipelines."
+    3="Explain the difference between stdout and stderr."
```

Comment:
Is there a missing LO about stdin?

Reply:
It's an interesting question...

I'm not _actually_ sure understanding stdin is super important, other than being able to construct a shell pipeline (where a very hand-waving understanding is probably sufficient).

I'm not sure we're ever going to have our trainees write programs which reads from stdin, and honestly in my real professional life I basically ever do. WDYT?